pkg/kubectl/cmd/util/factory.go

```diff
@@ -260,7 +253,8 @@ func NewFactory(optionalClientConfig clientcmd.ClientConfig) *Factory {
 				if err != nil {
 					return nil, err
 				}
-				return &clientSwaggerSchema{client, experimentalClient, api.Scheme}, nil
+				expClient, _ := expClients.Client()
```

Comment:
The experimental client failing to initialize is non-fatal. In clientSwaggerSchema we only use the experimentalClient if it is non-nil.